CONTRIBUTING.md

```diff
@@ -22,6 +23,30 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 kubectl scale deployments/inflate --replicas=3
 ```
 
+### Use local environment instead of GitHub Codespaces. 
+> Tested environment : WSL2
+
+1. **Install tools**
+   * make
+   * kubectl
+   * [helm](https://github.com/helm/helm/releases)
+   * [golang](https://go.dev/dl/) > 1.21
+   * [yq](https://github.com/mikefarah/yq/releases) / [jq](https://github.com/jqlang/jq/releases)
+   * [skaffold](https://skaffold.dev/docs/install/#standalone-binary)
+   * [azure-cli](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+
+2. **Provision cluster, build and deploy Karpenter** : Set `AZURE_SUBSCRIPTION_ID` to your subscription (and customize region in `Makefile-az.mk` if desired). Then run `make az-all`. This logs into Azure (follow the prompts), provisions AKS and ACR (using resource group and cluster name `$COMMON_NAME`, ACR it will not allow dashes), builds and deploys Karpenter, deploys sample `default` Provisioner and `inflate` Deployment workload.
+```
+az config set core.output=json
+export AZURE_SUBSCRIPTION_ID=xxxx-xxxx-xxxx-xxxx
+export COMMON_NAME=karpenter2test
+```
+
+3. Manually scale the `inflate` Deployment workload, watch Karpenter controller log and Nodes in the cluster. Example of manually scaling up to 3 pods:
+```
+kubectl scale deployments/inflate --replicas=3
```

Comment:
@tallaxes  thoughts on just having the inflate deployment be deployed with 3 replicas from the start in the inflate.yaml so that make `az-run-sample` will trigger scale up?

Reply:
I think it makes more sense to have the scale up be a follow up command folks can run and observe imo, rather that lumped into the larger make command.